services/azure/Compute/virtualMachines.yml

```diff
@@ -40,45 +47,79 @@ privileges:
     notes: >-
       Stops and releases allocation; VMs become unavailable until started.
   runCommand:
-    risks: [exfiltration:data, impact:hijack]
+    risks:
+      [
+        exfiltration:data,
+        exfiltration:crypto,
```

Comment:
I feel like anywhere we have `exfiltration` there is also `collection` risk.  I would still call out `execution` here since running a command seems like specifically what it's about

>[Adversaries may abuse PowerShell commands and scripts for execution](https://attack.mitre.org/techniques/T1059/001/)